api/advisory/db/models/tileserver.py

```diff
@@ -1,7 +1,7 @@
 """ We don't want alembic to manage the models in here!
 """
 
-from sqlalchemy import (Column, Integer, Date, String)
+from sqlalchemy import (Column, Integer, Date, String, Float)
```

Comment:
No need to touch this model - we should remove this model entirely, since there's no need to touch the tileserver database from the API.